packages/slate-editor/src/modules/editor/lib/table/insertTable.ts

```diff
@@ -10,7 +10,12 @@ export function insertTable(editor: Editor) {
         return;
     }
 
-    TableEditor.insertTable(editor, undefined, 3, 3);
+    TablesEditor.insertTable(editor, undefined, {
+        header: ['first_row'],
+        border: true,
+        rowsCount: 3,
+        columnsCount: 3,
+    });
```

Comment:
I've lifted the default table properties to the very top, to never apply them to a table created with default undefined properties.